test/functional/tests/fault_injection/test_fault_injection_opencas_load.py

```diff
@@ -0,0 +1,133 @@
+        output = TestRun.executor.run_expect_fail(cli.start_cmd(cache_dev=str(cache_part),
+                                                                cache_mode=str(cache_mode),
+                                                                force=False, load=False))
+        if output.stderr:
+            TestRun.LOGGER.info(output.stderr)
+        else:
+            TestRun.LOGGER.error(output.stdout)
+            TestRun.fail("Cache start succeeded (should fail)!")
+
+    with TestRun.step("Load cache."):
+        cache = casadm.load_cache(cache.cache_device)
+        caches_count = len(casadm_parser.get_caches())
+        if caches_count != 1:
+            TestRun.fail(f"Expected caches count: 1 Actual caches count: {caches_count}.")
+        cores_count = len(casadm_parser.get_cores(cache.cache_id))
+        if cores_count != 1:
+            TestRun.fail(f"Expected cores count: 1; Actual cores count: {cores_count}.")
+
+    with TestRun.step("Mount exported object."):
+        core.mount(mount_point)
+
+    with TestRun.step("Compare md5 sum of test file before and after loading cache."):
+        if test_file_md5_before != test_file.md5sum():
+            TestRun.LOGGER.error("Test file's md5 sum is different than before loading cache.")
+
+    with TestRun.step("Unmount exported object."):
+        core.unmount()
+
+    with TestRun.step("Stop cache."):
+        cache.stop()
+
+    with TestRun.step("Wipe filesystem from core device."):
+        core_part.wipe_filesystem()
+
+    with TestRun.step("Start cache with --force option."):
+        cache = casadm.start_cache(cache_part, cache_mode, force=True)
+
+    with TestRun.step("Add core device without filesystem."):
+        core = cache.add_core(core_part)
+
+    with TestRun.step("Fill exported object with data."):
+        dd = (Dd()
+              .input("/dev/zero")
+              .output(f"{core.system_path}")
+              .block_size(Size(1, Unit.Blocks4096))
+              .oflag("direct"))
+        dd.run()
+
+    with TestRun.step("Stop cache with option '--no-data-flush'."):
+        cache.stop(no_data_flush=True)
+        caches_count = len(casadm_parser.get_caches())
+        if caches_count != 0:
+            TestRun.fail(f"Expected caches count: 0; Actual caches count: {caches_count}.")
+
+    with TestRun.step("Try to start cache without loading metadata."):
+        output = TestRun.executor.run_expect_fail(cli.start_cmd(cache_dev=str(cache_part),
+                                                                cache_mode=str(cache_mode),
+                                                                force=False, load=False))
+        if output.stderr:
+            TestRun.LOGGER.info(output.stderr)
+        else:
+            TestRun.LOGGER.error(output.stdout)
+            TestRun.fail("Cache start succeeded (should fail)!")
+
+    with TestRun.step("Stop cache."):
+        casadm.stop_all_caches()
```

Comment:
This is not needed and it should never execute as even if cache had started in previous step, the test would have been immediately interrupted by `TestRun.fail()`.